src/pyhf/tensor/numpy_backend.py

```diff
@@ -34,6 +34,27 @@ def clip(self, tensor_in, min_value, max_value):
         """
         return np.clip(tensor_in, min_value, max_value)
 
+    def tile(self, tensor_in, repeats):
+        """
+        Repeat tensor data along a specific dimension
+
+        Example:
```

Comment:
To make `help()` and the docs as backend agnostic as possible can you also duplicate this example across the backends that you add `tile` to?

Reply:
Took care of this as I had some free cycles tonight.